actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -5,9 +5,9 @@ module Server
     class Configuration
       attr_accessor :logger, :log_tags
       attr_accessor :connection_class, :worker_pool_size
-      attr_accessor :redis, :channels_path
+      attr_accessor :channels_path
       attr_accessor :disable_request_forgery_protection, :allowed_request_origins
-      attr_accessor :url
+      attr_accessor :adapter, :cable, :url
```

Comment:
Agree.. I think `:adapter` here is vestigial. The "adapter" we care about lives inside `cable`.